test_crates/manifest_tests/README.md

```diff
@@ -0,0 +1,3 @@
+This subdirectory is for tests that use the Cargo.toml manifest as part of
+the test (for example, to test declared lint config), so
+`regenerate_test_rustdocs.sh` will not be run on test crates.
```

Comment:
I assume you confirmed that `regenerate_test_rustdocs.sh` on this branch succeeds without errors, continues to generate all the expected rustdoc files, and does not run on `manifest_tests`?

I think this is probably true, but best to be sure!

Reply:
:+1: It still works for me.  The script only goes through the top level of the test_crates directory to regenerate rustdocs, so by moving the new test into the manifest_tests directory, it doesn't run rustdoc.  And since manifest_tests doesn't contain child directories called `new` and `old`, it won't try to generate rustdocs for that folder either.